tests/cpp/multidevice.cpp

```diff
@@ -33,6 +42,34 @@ MultiDeviceTest::~MultiDeviceTest() {
   }
 }
 
+void MultiDeviceTest::waitForDebugger() {
+  char* rank_to_debug_str = getNvFuserEnv("MPI_DEBUG_RANK");
+  if (rank_to_debug_str == nullptr) {
```

Comment:
nit: I Ieave it up to you but I would find it more readable to have a test surrounding the call of `waitForDebugger`, i.e., inside test SetUp
```
if (getNvFuserEnv("MPI_DEBUG_RANK")) {
  waitForDebugger();
}
```